### PR TITLE
Adjusted values to match other profiles and corrected typo for 0,2mm AD5M Pro 0,4 Nozzle profile

### PR DESCRIPTION
**Pull request by mariuske**
A few changes to align the 0,2mm AD5M Pro 0,4 Nozzle profile:

1. Updated the "inner_wall_acceleration" parameter to match other profiles. If this was set to this value on purpose, please let me know and I will revert the change.
2. Corrected the default value for the "layer_height" to match the profile.
3. Updated the "initial_layer_speed" parameter to match the 0,2mm AD5M 0,4 Nozzle profile. 15 mm/s is too low.
4. Fixed a parameter typo. "internal_solid_fill_acceleration" => ""internal_solid_infill_acceleration"

@FlashforgeOfficial - Please review these changes and let me know if you would like me to change anything. I also found the following mismatches for the 0,2mm AD5M Pro 0,4 Nozzle profile, please confirm if these are correct or if I should update them as well:

1. The "bottom_shell_layers" is set to "5" for this profile, but "3" for the others. Was this on purpose?
2. Skirt ("skirt_loops" parameter) is only activated for this profile, but not for other profiles. Was this on purpose?
3. The parameter "wipe_speed" is not set for this profile, but is set to"200" for other profiles. Was this on purpose?

### CODE REVIEW

**Review by FlashforgeOfficial — APPROVED** [review]
@mariuske 
Right, I realized that I forgot to recheck and update these changes for this 0.2mm profile for  AD5M pro 0.4 nozzle. 
All the first four and additional three mismatches from other profile are welcome to be updated.  
Good job! 👍

**Review by FlashforgeOfficial — APPROVED** [review]

**Comment by FlashforgeOfficial** [comment]
@mariuske I just made the approval, seems that it requires one more review for the merging these changes. 

image

**Comment by mariuske** [comment]
Great, thanks for reviewing. I'll make another commit for the remaining changes.

**Comment by mariuske** [comment]
All values in this profile are now aligned and I believe this forms a good basis for further testing and tweaking of the profiles.

@SoftFever - This PR is now ready to be merged.

**Review by SoftFever — APPROVED** [review]

File: resources/profiles/Flashforge/process/0.20mm Standard @Flashforge AD5M Pro 0.4 Nozzle.json [hunk 1]
```diff
@@ -9,10 +9,10 @@
   "reduce_crossing_wall": "0",
   "max_travel_detour_distance": "0",
   "bottom_surface_pattern": "monotonic",
-  "bottom_shell_layers": "5",
+  "bottom_shell_layers": "3",
   "bottom_shell_thickness": "0",
   "bridge_flow": "1",
-  "bridge_speed": "25",
+  "bridge_speed": "50",
```
Comment (SoftFever) [inline resources/profiles/Flashforge/process/0.20mm Standard @Flashforge AD5M Pro 0.4 Nozzle.json:15]: I suggest to have 25 as default for best bridge performance though

Comment (mariuske) [reply resources/profiles/Flashforge/process/0.20mm Standard @Flashforge AD5M Pro 0.4 Nozzle.json:15]: Ok, noted. I will revert the change for this value in a new commit. This was done to align with the other Flashforge profiles, should it be updated to 25 for all profiles?

Comment (mariuske) [reply resources/profiles/Flashforge/process/0.20mm Standard @Flashforge AD5M Pro 0.4 Nozzle.json]: I have now updated the bridge speed value to 25 for all profiles.

**Comment by mariuske** [comment]
The bridge speed has now been updated as per suggestion. I also tweaked the initial layer acceleration value for all profile to match a more common practise based on other profiles and recommandtions from filament producers (e.g. eSUN).

@SoftFever - Please review and merge the PR if you're fine with the changes.

**Comment by SoftFever** [comment]
> The bridge speed has now been updated as per suggestion. I also tweaked the initial layer acceleration value for all profile to match a more common practise based on other profiles and recommandtions from filament producers (e.g. eSUN).
> 
> @SoftFever - Please review and merge the PR if you're fine with the changes.


Cool, thanks.

I don't have hands-on experience with these machines, so my insights come from general experience. 
Feel free to update or tweak the parameters since you're using them. 
As a rule of thumb, built-in parameters should be slightly conservative.

**Review by SoftFever — APPROVED** [review]

**Comment by SoftFever** [comment]
Thanks for the work.
Merged

**Pull request merged**